odoo/addons/base/migrations/11.0.1.3/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,428 @@
+base         / report.paperformat       / margin_right (float)          : previously in module report
+base         / report.paperformat       / margin_top (float)            : previously in module report
+base         / report.paperformat       / name (char)                   : previously in module report
+base         / report.paperformat       / orientation (selection)       : previously in module report
+base         / report.paperformat       / page_height (integer)         : previously in module report
+base         / report.paperformat       / page_width (integer)          : previously in module report
+base         / report.paperformat       / report_ids (one2many)         : previously in module report
+# Done: Reassign module from 'report' to 'base'
+
+base         / res.bank                 / fax (char)                    : DEL 
+base         / res.company              / custom_footer (boolean)       : DEL 
+# DONE: pre-migration: rename column
+
+base         / res.company              / external_report_layout (selection): NEW selection_keys: ['background', 'boxed', 'clean', 'standard']
+# NOTHING TO DO
+
+base         / res.company              / font (many2one)               : DEL relation: res.font
+# DONE: pre-migration: rename column
+
+base         / res.company              / paperformat_id (many2one)     : previously in module report
+# Done: Reassign module from 'report' to 'base'
+
+base         / res.company              / report_footer (text)          : NEW 
+base         / res.company              / report_header (text)          : NEW 
+# NOTHING TO DO
+
+base         / res.company              / rml_footer (text)             : DEL 
+base         / res.company              / rml_header (text)             : DEL required: required, req_default: function
+base         / res.company              / rml_header1 (char)            : DEL 
+base         / res.company              / rml_header2 (text)            : DEL required: required, req_default: function
+base         / res.company              / rml_header3 (text)            : DEL required: required, req_default: function
+base         / res.company              / rml_paper_format (selection)  : DEL required: required, selection_keys: ['a4', 'us_letter'], req_default: function
+# DONE: pre-migration: rename column
+
+base         / res.country              / address_view_id (many2one)    : NEW relation: ir.ui.view
+base         / res.country              / name_position (selection)     : NEW selection_keys: ['after', 'before']
+base         / res.country              / vat_label (char)              : NEW 
+base         / res.currency             / currency_subunit_label (char) : NEW 
+base         / res.currency             / currency_unit_label (char)    : NEW 
+# NOTHING TO DO
+
+base         / res.currency.rate        / name (datetime)               : type is now 'date' ('datetime')
+# DONE: pre-migration: copy column
+# DONE: pre-migration: change field type
```

Comment:
I can't find where this happens. Am I missing something?

Reply:
:) your are right. Its not done yet.